src/components/Text/Text.tsx

```diff
@@ -47,9 +47,9 @@ export interface TextProps {
    */
   as?: string;
   /**
-   * The ref to the html dom element, it can be an anchor or a button
+   * The ref to the HTML DOM element.
    */
-  ref?: React.Ref<HTMLParagraphElement>;
+  ref?: React.Ref<any>;
```

Comment:
This cannot be a `Ref<HTMLParagraphElement>` because Blockquote extends it and is a `HTMLQuoteElement`.

It couldn't be set as `Ref<HTMLElement>` either because of a TS compiler error about conflicting props. I've set it as `any` for now, just like we did for the Anchor component.